war/src/main/java/io/lumeer/engine/controller/SecurityFacade.java

```diff
@@ -310,22 +316,22 @@ private void removeViewRole(String projectId, int viewId, String user, String gr
 
    private Map<String, Object> projectFilter(String projectId) {
```

Comment:
`projectId` parameters in such methods should be renamed to `projectCode` not to confuse anybody in the future.